### PR TITLE
docs(updated-setup)

### PR DESCRIPTION
**Pull request by crisgarner**
Added some updates to versions and added the improved rewording and explanations by @daniel0ar in https://github.com/privacy-scaling-explorations/maci-platform/pull/358 🙏

### CODE REVIEW

File: docs/01_setup.md [hunk 1]
```diff
@@ -145,13 +152,13 @@ cd apps/subgraph
...
 ```
 
-4. Run `pnpm run build`. You can use env variables `NETWORK` and `VERSION` to switch config files.
-   - Create an `.env` file, and run the follow command in the console `export $(xargs < .env)`
-5. Run `graph auth --studio {key}`. You can find the key in subgraph studio dashboard.
-6. Run `pnpm run deploy` to deploy subgraph
-
-> [!IMPORTANT]
-> The `pnpm run deploy` command call `maci-subgraph` as the subgraph name as default, but if you named your subgraph differently (e.g. maci-graph, my-graph, etc.), please change the command to `graph deploy --node https://api.studio.thegraph.com/deploy/ :your_subgraph_name`
+4. If you have not yet installed the graph CLI in your computer globally do so by running `pnpm i -g @graphprotocol/graph-cli`
```
Comment (kittybest) [inline docs/01_setup.md:155]: Maybe change to this:
If you haven't installed the graph CLI in your computer globally yet, do so by running `pnpm i -g @graphprotocol/graph-cli`.

File: docs/01_setup.md [hunk 2]
```diff
@@ -145,13 +152,13 @@ cd apps/subgraph
...
+5. Run `pnpm run build`. You can use env variables `NETWORK` and `VERSION` to switch config files.
+   - Create an `.env` file, and run the following command in the console `export $(xargs < .env)`
+6. Copy the auth command from the Subgraph Studio graph details in section "Auth and Deploy", paste it into the terminal and execute it. I should look something like this `graph auth --studio {key}`.
```
Comment (kittybest) [inline docs/01_setup.md:158]: Maybe change to this:
Copy the authenticate command from the 'Auth and Deploy' section in the Subgraph Studio graph details, paste it into the terminal, and execute it. The command should look like this: `graph auth --studio {key}`.

File: docs/01_setup.md [hunk 3]
```diff
@@ -145,13 +152,13 @@ cd apps/subgraph
...
+7. Change `package.json` deploy script to graph deploy --node https://api.studio.thegraph.com/deploy/<name> where <name> is the name that you gave to the subgraph you created.
```
Comment (kittybest) [inline docs/01_setup.md:159]: The command is 
```bash
graph deploy --node https://api.studio.thegraph.com/deploy/ <name>
```
there is a space between `{blah blah blah...}/deploy/` and `<name>`

**Review by kittybest — COMMENTED** [review]
Thanks! just left some comment!

**Comment by crisgarner** [comment]
Thanks! added the recommendations.

File: docs/01_setup.md [hunk 4]
```diff
@@ -92,7 +105,7 @@ cp default-deployed-contracts.json deployed-contracts.json
 ```
 
 > [!IMPORTANT]
-> Make sure that you use the production zkeys, and set the `pollDuration` with the correct time on **seconds**.
+> Maybe change to this: Ensure that you replace the `coordinatorPubKey` generated in the [previous step](#Generate-MACI-Keys) and set the `pollDuration` to the correct time in **seconds**.
```
Comment (kittybest) [inline docs/01_setup.md:108]: Remove the "Maybe change to this:" 😂

File: docs/01_setup.md [hunk 5]
```diff
@@ -109,7 +122,7 @@ Of course you could run without the `incremental` flag to deploy everything by y
 pnpm deploy:NETWORK
 ```
 
-Run pnpm deploy-poll to deploy your first Poll (you can specify the network by appending :network to the command, e.g. pnpm deploy-poll:sepolia - please refer to the available networks on the `package.json` scripts section).
+Deploy your first Poll (you need specify the same network as the MACI contracts by appending :network to the command, e.g. pnpm deploy-poll:sepolia - please refer to the available networks on the `package.json` scripts section).
```
Comment (kittybest) [inline docs/01_setup.md:125]: typo: you need "to" specify

File: docs/01_setup.md [hunk 6]
```diff
@@ -145,13 +158,16 @@ cd apps/subgraph
...
+5. Run `pnpm run build`. You can use env variables `NETWORK` and `VERSION` to switch config files.
+   - Create an `.env` file, and run the following command in the console `export $(xargs < .env)`
+6. Copy the authenticate command from the 'Auth and Deploy' section in the Subgraph Studio graph details, paste it into the terminal, and execute it. The command should look like this: `graph auth --studio {key}`.
+7. Change `package.json` deploy script to `graph deploy --node https://api.studio.thegraph.com/deploy/ <name>` where <name> is the name that you gave to the subgraph you created.
+8. Run `pnpm run deploy` to deploy the subgraph.
+9. Note down the url of the deployed subgraph API endpoint.
 
 > [!IMPORTANT]
-> The `pnpm run deploy` command call `maci-subgraph` as the subgraph name as default, but if you named your subgraph differently (e.g. maci-graph, my-graph, etc.), please change the command to `graph deploy --node https://api.studio.thegraph.com/deploy/ :your_subgraph_name`
+> If you always use the latest version the URL will be `https://api.studio.thegraph.com/query/[some number]/[name]/version/latest`.
```
Comment (kittybest) [inline docs/01_setup.md:170]: I checked out what does the number mean, it's the `USER_ID`, so this URL should be
`https://api.studio.thegraph.com/query/<USER_ID>/<YOUR_SUBGRAPH_NAME>/version/latest`

File: docs/01_setup.md (hunk 4 above)
Comment (crisgarner) [reply docs/01_setup.md:108]: whoops 😆

**Review by kittybest — APPROVED** [review]
thanks a lot! but please fix the commitlint issue :)

**Pull request merged**